### ISSUE TITLE
input window should grow vertically as input wraps

### ISSUE THREAD

**Issue by ejelly** [glowing-bear/glowing-bear#794]
The input field currently has a fixed vertical size, which means that long input is not always fully visible. This applies to multi-line input as well as input with lines longer that the input field's width.

The input field should always resize to contain the full input without scrolling.

**Comment by lorenzhs** [comment]
Good idea. People who don't want this can always override the input bar's height with Custom CSS (that feature is really beginning to pay off!)

**Comment by Transfusion** [comment]
The input bar height is defined as an absolute value, `35px`, in three different places:

https://github.com/glowing-bear/glowing-bear/blob/6508993fea5e4bd2a00f5809433ca9e2c914881a/src/css/glowingbear.css#L346-L357

https://github.com/glowing-bear/glowing-bear/blob/6508993fea5e4bd2a00f5809433ca9e2c914881a/src/css/glowingbear.css#L66-L70

https://github.com/glowing-bear/glowing-bear/blob/6508993fea5e4bd2a00f5809433ca9e2c914881a/src/css/glowingbear.css#L394-L403

A complicating issue is that the footer, which contains the input field, has `position: fixed;`, than say, if it were in the same nesting level as the buffer container.

For a larger input field in the meantime, use this as custom CSS:
```css
#bufferlines { bottom: 10em; }
#sendMessage, .footer { height: 10em; }
```
